docker-bake.hcl

```
@@ -0,0 +1,218 @@
+group "all" {
+  targets = [
+    "bitcoin-28",
+    "bitcoin-27",
+    "bitcoin-26",
+    "v0-21-1",
+    "v0-20-0",
+    "v0-19-2",
+    "v0-17-0",
+    "v0-16-1",
+    "bitcoin-unknown-message",
+    "bitcoin-invalid-blocks",
+    "bitcoin-50-orphans",
+    "bitcoin-no-mp-trim",
+    "bitcoin-disabled-opcodes",
+    "bitcoin-5k-inv"
+  ]
+}
+
+group "maintained" {
+  targets = [
+    "bitcoin-28",
+    "bitcoin-27",
+    "bitcoin-26"
+  ]
+}
+
+group "practice" {
+  targets = [
+    "bitcoin-unknown-message",
+    "bitcoin-invalid-blocks",
+    "bitcoin-50-orphans",
+    "bitcoin-no-mp-trim",
+    "bitcoin-disabled-opcodes",
+    "bitcoin-5k-inv"
+  ]
+}
+
+group "vulnerable" {
+  targets = [
+    "v0-21-1",
+    "v0-20-0",
+    "v0-19-2",
+    "v0-17-0",
+    "v0-16-1",
+  ]
+}
+
+target "maintained-base" {
+  dockerfile = "./Dockerfile"
+  context = "./resources/images/bitcoin"
+  args = {
+    REPO = "bitcoin"
```

Shouldn't this be as shown below?
```suggestion
    REPO = "bitcoin/bitcoin"
```